app/styles/components/_detail-learning-materials.scss

```diff
@@ -1,11 +1,23 @@
-.detail-learning-materials {
-  .learning-material-actions {
-    @include span-columns(3 of 12);
-    @include omega;
-    float: right;
+.detail-block {
```

Comment:
Why did you un-scope this?  That will cause this style to creep out into the rest of the app.  If that should happen then this belongs in detailView.scss